AardvarkSample/AardvarkSample/AppDelegate.swift

```diff
@@ -53,4 +57,11 @@ class SampleAppDelegate: UIResponder, UIApplicationDelegate {
     func applicationWillTerminate(_ application: UIApplication) {
         log("\(#function)", type: .error)
     }
+
+}
+
+// MARK: - Uncaught Exception Handling
+
+func existingUncaughtExceptionHandler(exception: NSException) {
+    print("Existing uncaught exception handler got called")
```

Comment:
should we print the exception here?

this is super nifty btw

Reply:
Sure 👍